lib/govuk_app_config/govuk_xray.rb

```diff
@@ -0,0 +1,28 @@
+require 'aws-xray-sdk/facets/rack'
+require 'aws-xray-sdk/facets/rails/ex_middleware'
+
+module GovukXRay
+  def self.initialize(app)
+    app.middleware.insert 0, XRay::Rack::Middleware
+    app.middleware.use XRay::Rails::ExceptionMiddleware
+  end
+
+  def self.start
```

Comment:
I went for a different name because it doesn't just set configuration, it also starts the recorder. 
 And unlike `GovukLogging` it's called in the `after_initialize` bit of the railtie.

Reply:
That's true, I don't mind either way.